webrtc.html

```diff
@@ -2871,6 +2929,242 @@
+
+          <dd>
+            This event handler, of event handler event
+            type <code><a href=
+            "#event-dtlstransport-statechange">statechange</a></code>,
+            MUST be fired any time the <a href=
+            "#dom-dtlstransport-state"><code><a>RTCDtlsTransport</a></code>
+            state</a> changes.
+          </dd>
+        </dl>
+
+        <h3>RTCDtlsTransportState Enum</h3>
+
+        <dl class="idl" title="enum RTCDtlsTransportState">
+          <dt>new</dt>
+
+          <dd>DTLS has not started negotiating yet.</dd>
+
+          <dt>connecting</dt>
+
+          <dd>DTLS is in the process of negotiating a secure connection.</dd>
+
+          <dt>connected</dt>
+
+          <dd>DTLS has completed negotiation of a secure connection.</dd>
+
+          <dt>closed</dt>
+
+          <dd>The transport has been closed.</dd>
+
+          <dt>failed</dt>
+
+          <dd>The transport has failed as the result of an error (such
+          as a failure to validate the remote fingerprint).</dd>
+        </dl>
+      </section>
+
+      <section>
+        <h3>RTCIceTransport Interface</h3>
+
+        <p>The <code><a>RTCIceTransport</a></code> interface allows an
+        application access to information about the ICE transport over
+        packets are sent and received.  In particular, ICE manages
+        peer-to-peer connections which involve state which the
+        application may want to access.</p>
+
+        <dl class="idl" title=
+          "interface RTCIceTransport">
+          <dt>readonly attribute RTCIceRole role</dt>
+
+          <dd>
+            <p>The <dfn id=
+            "dom-icetransport-role"><code>role</code></dfn> attribute
+            MUST return the ICE role of the transport.
+          </dd>
+
+          <dt>readonly attribute RTCIceComponent component</dt>
+
+          <dd>
+            <p>The <dfn id=
+            "dom-icetransport-component"><code>component</code></dfn>
+            attribute MUST return the ICE component of the transport.
+          </dd>
+
+          <dt>readonly attribute RTCIceConnectionState state</dt>
+
+          <dd>
+            <p>The <dfn id=
+            "dom-icetransport-state"><code>state</code></dfn>
+            attribute MUST return the state of the transport.
+          </dd>
+
+          <dt>readonly attribute RTCIceGatheringState gatheringState</dt>
+
+          <dd>
+            <p>The <dfn id=
+            "dom-icetransport-gatheringstate"><code>gathering
+            state</code></dfn> attribute MUST return the gathering
+            state of the transport.
+          </dd>
+
+          <dt>sequence&lt;RTCIceCandidate&gt; getLocalCandidates()</dt>
+
+          <dd>
+            <p>An array containing the local ICE candiates gathered
+            for this <code><a>RTCIceTransport</a></code> and sent in
+            <code><a href=
+            "#dom-peerconnection-onicecandidate">PeerConnection.onicecandidate</a></code></p>
```

Comment:
Should be reworded to not look like a static method, I think (there was a recent sweep of this).